.github/dependabot.yml

```diff
@@ -0,0 +1,15 @@
+version: 2
+updates:
+  - package-ecosystem: "gradle"
```

Comment:
Hmm, it seems dependabot cannot process `build.gradle.kts` files at the moment
I'll mark as draft